Monika After Story/game/zz_selector.rpy

```diff
@@ -3680,7 +3734,19 @@ label monika_ribbon_select:
             ):
                 use_acs.pop(index)
 
-        # make sure ot use ribbon for remover type
+        # generate acs_type_list
+        acs_type_list = []
+        for item in use_acs:
+            if ( item.get_sprobj().acs_type not in acs_type_list ) and (
+                    store.mas_sprites.is_hairacs_compatible(
+                        monika_chr.hair,
+                        item.get_sprobj()
+                    )
+                ):
+                    acs_type_list.append(item.get_sprobj().acs_type)  
```

Comment:
Looking at this, I think there might be a way we can do this where we can have more control over the button names and what elements are part of each category. 

If we use a dict with the key as the button name and the value as a list of selectables associated wtih that key, then in the screen we can check if a button is selected and only show the selectables for that key. (and if no buttons are selected, show the full item list). We can pass this dict into the selector labels.

For generic ACS type cases (like this case, for example), we can have a function in the `mas_selspr` store that does the mapping based on ACS type, and another function for adjusting the keys so they are appropriate for visuals.

This way would give us more flexibility in which buttons to show and which selectables to categorize under a button, (including the option of having a selectable be under multiple buttons), which will help with making this work with hair/clothes selectors.

Reply:
I have added a new commit that hopefully addresses this